package.json

```diff
@@ -58,6 +58,8 @@
         "whatwg-fetch": "^3.0.0"
     },
     "dependencies": {
+        "@types/d3": "^5.7.2",
```

Comment:
i think the @types entry can go in `devDependencies`, since typings aren't technically required to build the code

Reply:
oops, I tagged github user [@]types 😆

Reply:
aha, did not know that